Zend/tests/grammar/regression_004.phpt

```diff
@@ -9,6 +9,6 @@ class Obj
     function return(){} // valid
 }
 
-function echo(){} // not valid
---EXPECTF--
-Parse error: syntax error, unexpected 'echo' (T_ECHO), expecting %s in %s on line 9
+function echo(){} // valid
```

Comment:
Maybe an error like "Can not redeclare intrinsic PHP construct `echo`..." makes more sense here? Same for `exit`, `die` and `print` ?